lib/v2/gocn/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'gocn.vip': {
+        _name: 'GoCN',
+        '.': [
+            {
+                title: 'GoCN',
+                docs: 'https://docs.rsshub.app/programming.html#GoCN',
```

Comment:
```suggestion
                title: '文章',
                docs: 'https://docs.rsshub.app/programming.html#GoCN',
                source: ['/'],
                target: '/gocn',
```
Title should match H3 heading in the docs